package.json

```diff
@@ -18,21 +18,25 @@
     ]
   },
   "scripts": {
-    "build": "babel src/ -d build/",
+    "build": "babel src/ -d build/ --extensions \".js,.ts\" --source-maps inline",
+    "build:types": "tsc --emitDeclarationOnly",
     "prepublish": "yarn run build",
```

Comment:
You OK with chaining them?
```suggestion
     "prepublish": "yarn run build && yarn run build:types",
```

Reply:
I noticed that there is `index.d.ts` in the project root.  This seems to have type definitions for all the public types in the project and I didn't want to double up.  Maybe when the entire codebase is TS, then include the built definitions?  What do you think?

Also I can't see what steps are performed for a release - I presume the types are added manually?

Reply:
> Maybe when the entire codebase is TS, then include the built definitions? What do you think?

I prefer that, since we want to stick to our index.d.ts until the whole project and all types have been converted.

Reply:
> Also I can't see what steps are performed for a release - I presume the types are added manually?

Ah I see that the typings are referenced in the `package.json`, which is probably how that works.